_sass/components/_section.scss

```diff
@@ -28,7 +28,7 @@
 		p {
 			font-family: _font(family-heading);
 			font-size: 1em;
-			font-weight: _font(weight-heading-alt);
+			font-weight: _font(weight);
 			letter-spacing: _font(kerning-heading);
```

Comment:
`0.03em` is not the right `letter-spacing` we should be using, it should be not declared for `p`.

Reply:
I will remove it but these rules only apply to "section header p", _p_ inside _header_ inside _section_, and that never happens anywhere. Probably the whole block should be removed?